src/test/java/org/symphonyoss/symphony/messageml/elements/form/TimezonePickerTest.java

```diff
@@ -0,0 +1,248 @@
+    String input = "<messageML><form id=\"" + formId + "\">" +
+            "<timezone-picker " +
+            "title=\"This is a \\n timezone picker\"/>" +
+            ACTION_BTN_ELEMENT + "</form></messageML>";
+
+    expectedException.expect(InvalidInputException.class);
+    expectedException.expectMessage("The attribute \"name\" is required");
+    context.parseMessageML(input, null, MessageML.MESSAGEML_VERSION);
+  }
+
+  @Test
+  public void testTimezonePickerWrongValueAttr() throws Exception {
+    String input = "<messageML><form id=\"" + formId + "\">" +
+            "<timezone-picker " +
+            "name=\"timezone-name\" " +
+            "value=\"America/Rome\"/>" +
+            ACTION_BTN_ELEMENT + "</form></messageML>";
+
+    expectedException.expect(InvalidInputException.class);
+    expectedException.expectMessage("Attribute \"value\" contains an invalid timezone: America/Rome");
+    context.parseMessageML(input, null, MessageML.MESSAGEML_VERSION);
+  }
+
+  @Test
+  public void testTimePickerWrongDisabledTimezone() throws Exception {
+    String input = "<messageML><form id=\"" + formId + "\">" +
+            "<timezone-picker " +
+            "name=\"timezone-name\" " +
+            "disabled-timezone='[\"America/Detroit\", \"America/Paris\"]'/>" +
+            ACTION_BTN_ELEMENT + "</form></messageML>";
+
+    expectedException.expect(InvalidInputException.class);
+    expectedException.expectMessage("Attribute \"disabled-timezone\" contains an invalid timezone: America/Paris");
+    context.parseMessageML(input, null, MessageML.MESSAGEML_VERSION);
+  }
+
+  @Test
+  public void testTimePickerDisabledTimezoneNotArray() throws Exception {
+    String input = "<messageML><form id=\"" + formId + "\">" +
+            "<timezone-picker " +
+            "name=\"timezone-name\" " +
+            "disabled-timezone='\"America/Detroit\", \"America/Los_Angeles\"'/>" +
+            ACTION_BTN_ELEMENT + "</form></messageML>";
+
+    expectedException.expect(InvalidInputException.class);
+    expectedException.expectMessage("Error parsing json in attribute \"disabled-timezone\". " +
+            "It should contain an array of Strings");
+    context.parseMessageML(input, null, MessageML.MESSAGEML_VERSION);
+  }
+
+  @Test
+  public void testEmptyMarkdown() throws Exception {
+    String input = "<messageML><form id=\"" + formId + "\">" +
+            "<timezone-picker " +
+            "name=\"timezone-name\"/>" +
+            ACTION_BTN_ELEMENT + "</form></messageML>";
+    String EXPECTED_MARKDOWN = "Form (log into desktop client to answer):\n"
+            + "---\n"
+            + "(Timezone Picker)(Button:Send)\n"
+            + "---\n";
+    context.parseMessageML(input, null, MessageML.MESSAGEML_VERSION);
+    assertEquals("Markdown", EXPECTED_MARKDOWN, context.getMarkdown());
+  }
+
+  @Test
+  public void testMarkdownWithUnderscore() throws Exception {
+    String input = "<messageML><form id=\"" + formId + "\">" +
```

Comment:
I don't if you are familiar with language injection in Intellij (https://www.jetbrains.com/help/idea/using-language-injections.html#use-language-injection-comments)
maybe it can help to edit HTML like content (in a separate editor) although sometimes it is a bit buggy

Reply:
Also I just wish we had multiline strings in Java already (Java 15....)

Reply:
wow that could be really helpful indeed. Thank you!